server/monitor.go

```diff
@@ -363,6 +380,12 @@ func (s *Server) Connz(opts *ConnzOptions) (*Connz, error) {
 				if mqttCID != _EMPTY_ && client.getMQTTClientID() != mqttCID {
 					continue
 				}
+
+				// Do client kind filtering
+				if kind != _EMPTY_ && client.kindString() != kind {
```

Comment:
It may be worth `strings.ToLower`-ing both `client.kindString()` and `kind` to add case-insensitivity for those who are crafting their own API calls.

Reply:
Great! Will add this in shortly

Reply:
Can we get this one revived and merged in to main?

Reply:
Yup. I can take a look at this now